daktari/__main__.py

```diff
@@ -18,6 +18,11 @@ def main() -> int:
     args = argument_parser.parse_args()
     if args.debug:
         logging.basicConfig(level=logging.DEBUG)
+
+    if args.generate:
+        write_local_config_template()
+        return 0
+
```

Comment:
What happens if the file already exists?

Reply:
It'll overwrite it. I could make it warn or something but didn't feel worth the effort atm. I guess if it becomes a more sophisticated file at some point in the future we might want it to merge changes or something but feels YAGNI for now.